content/post/multimedia/elzero-web-school/index.ar.md

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
This has already been added.
You need to check out from the `main` branch after you pull the newest updates.